README.md

```diff
@@ -6,6 +6,21 @@ Cerberus watches the Kubernetes/OpenShift clusters for dead nodes, system compon
 ### Workflow
 ![Cerberus workflow](media/cerberus-workflow.png)
 
+### Cerberus as a Python Package
+Cerberus is avialable as a python package to ease the installation and setup.
+
+#### Installation
+To install the lastest release:
+
+`pip3 install cerberus-client`
+
+#### Usage
+To start cerberus monitoring, execute:
+
+`cerberus_client -c <config_file_location>`
+
+**NOTE**: When config_file_location is not passed, default config file is used.
+
 ### Install the dependencies
 ```
 $ pip3 install -r requirements.txt
```

Comment:
Do we need to still install dependencies if we use the pip instal cerberus-client. I think that we should maybe specify you can use the python and do these new steps you added or clone the repo and you will have to install the requirements. Thoughts?

Reply:
No, we need not install dependencies, it's automatically installed.

Reply:
Yes, ok! The way it's currently laid out is a little confusing that it still says to install dependencies.

I think adding in a title just saying 2 possibilities of installation.

Reply:
+1 for adding that there are two ways to install and use cerberus.

Reply:
Changed it. Thank you.

Reply:
Looks awesome! Thanks